maro/simulator/scenarios/vm_scheduling/business_engine.py

```diff
@@ -114,7 +118,13 @@ def _load_configs(self):
 
         self._delay_duration: int = self._config.DELAY_DURATION
         self._buffer_time_budget: int = self._config.BUFFER_TIME_BUDGET
-        self._pm_amount: int = self._config.PM.AMOUNT
+        # Oversubscription rate.
+        self._max_cpu_oversubscription_rate: float = self._config.MAX_CPU_OVERSUBSCRIPTION_RATE / 100
```

Comment:
If the float rate is used in BE, why not directly define the unit and type of these RATEs?